x-pack/plugins/triggers_actions_ui/server/data/lib/time_series_query.ts

```diff
@@ -59,9 +58,8 @@ export async function timeSeriesQuery(
       },
       // aggs: {...}, filled in below
     },
-    ignoreUnavailable: true,
-    allowNoIndices: true,
-    ignore: [404],
```

Comment:
Is this `ignore: [404]` kind of processing handled by the new ES client?  Since it's not in the "replacement" options below.  I think we use this with the old client to ensure we get back 0 hits rather than get a 404 which we would treat as an error, but would need to poke around a bit more on it.  LMK if you want me to ...

I noticed it's also missing from some other API calls made in this plugin, so guessing it must be handled already by the client ...

Reply:
Good catch. The params structure was changed. Will fix it! 👍